lib/src/components/Sections/index.vue

```diff
@@ -1403,7 +1414,7 @@ export default {
 	}
   },
   mounted() {
-    if(this.sectionsError !== "") {
+    if(this.sectionsError !== "" && !this.registeredPage(this.errorResponseStatus === 404 ? 'page_not_found' : 'project_not_found')) {
```

Comment:
why don't you actually rely on the `section_pre_render` hook to select the page_not_found component (and thus leave this management to the host project instead of doing a specific management in the library) ?